### DISCUSSION TITLE
How to Settle the expenses when the amount is refunded

### DISCUSSION THREAD

**Discussion by abhishek199-dhn** [ananthakumaran/paisa#220] [Q&A]
I want to settle a refund so that the expense zeros This should cancel out the original expense when recording the refund. What's the best way to record these transactions?

For example:

**Initial Purchase:**
Record the purchase of the fridge as an expense:
```
2024/04/23 Bought Fridge from Amazon
    Expenses:Shopping:Fridge    20000.00 INR
    Assets:Checking:HDFC_Savings
```    

**Refund**
Record the refund to cancel out the expense
```
2024/04/25 Refund for Fridge
    Expenses:Shopping:Fridge    -20000.00 INR
    Assets:Checking:HDFC_Savings
```

This displays an error in the doctor 
```

Expense Account should never have debit entry.

-20000.00 got debited from Expenses:Shopping:Fridge on 25 March 2024

```

**Comment by ananthakumaran** [comment]
check https://github.com/ananthakumaran/paisa/discussions/92

**Reply by abhishek199-dhn** [reply]
Thanks!
